components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -69,9 +69,10 @@ To delete a Data Feed instance in Chef Automate:
 The settings in `config.toml` apply across all configured Data Feed instances.
 {{< /info >}}
 
-Modify Data Feed behavior with configuration settings in `config.toml`.
+Modify Data Feed behavior with configuration settings.
+
+1. Create a patch file e.g. data-feed-patch.toml and patch the configuration using the Chef Automate command-line tool.
```

Comment:
@satellite15 What would be the cli command to create a patch file? We can include this in the doc

Reply:
There is no cli command to create a patch file. You manually create it based on the example.

Reply:
"Create a patch file" is a separate step from "Patch the configuration" with the command listed in line 87?